src/styles/blocks/container.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+  margin: 0 auto;
+  max-width: 1020px;
```

Comment:
Consider using a variable for the max-width value to maintain consistency and easy updates in the future.